### DISCUSSION TITLE
How to deselect all entries in a 2many relationship

### DISCUSSION THREAD

**Discussion by hochstibe** [aminalaee/sqladmin#551] [Q&A] [answered]
For example users and groups are connected with a many2many relationship.
When creating or editing a user, I can select or deselect the groups.

However, I cant deselect all groups (at least one is still active). 

The same is for the one2many relationships (for example users to adresses on your demo page)

Do I miss something obvious? I can provide a minimal example, if required

**Comment by aminalaee** [comment, marked as answer]
I think a minimal example would be nice, but for M2M we just have HTML <select multiple> so using CTRL + click you can select or deselect as you want.
But for the One-To-Many, I think it should be nullable and what happens is we create this select:

<select>
<option></option>
<option>A</option>
<option>A</option>
</select>

So any empty one is added automatically so you can select that to deselect.

**Reply by hochstibe** [reply]
Ctrl + click works. I thought I tried this... Thanks a lot for the quick reply and the great project :-)
